src/instructlab/config.py

```diff
@@ -161,6 +168,7 @@ def get_default_config():
 
 def read_config(config_file=DEFAULT_CONFIG):
     """Reads configuration from disk."""
+    # pylint: disable=W0108
```

Comment:
Where is the unnecessary lambda?   Don't see need to add this pylint disable

Reply:
sorry, left over. Will remove thank for catching